internal/pkg/aws/ecs/ecs.go

```diff
@@ -342,9 +345,23 @@ func (e *ECS) ActiveClusters(arns ...string) ([]string, error) {
 }
 
 // ActiveServices returns the subset of service arns that have an ACTIVE status.
```

Comment:
Can we add a comment to suggest that this function should only be called on services in the same cluster?